app/lib/package/backend.dart

```diff
@@ -1575,6 +1575,24 @@ class PackageBackend {
       current.gcpLock = GcpPublishingLock(oauthUserId: agent.payload.sub);
     }
   }
+
+  // ignore: deprecated_member_use_from_same_package
+  /// Migrates the deprecated [Package.automatedPublishingJson] JSON field
+  /// and makes sure that the only related value is in the
+  /// [Package.automatedPublishingField].
+  Future<void> backfillAutomatedPublishingField() async {
+    await for (final p in db.query<Package>().run()) {
+      // ignore: deprecated_member_use_from_same_package
+      if (p.automatedPublishingJson == null) {
+        continue;
+      }
+      await withRetryTransaction(db, (tx) async {
+        final pkg = await tx.lookupValue<Package>(p.key);
+        pkg.automatedPublishing = pkg.automatedPublishing;
```

Comment:
Shouldn't this be:
```suggestion
        pkg.automatedPublishing = pkg.automatedPublishingJson;
```

Reply:
The `automatedPublishing` field has already a logic to access the json content if available, and that's the consolidated view the users are seeing.

Reply:
Ah! nice